easybuild/easyconfigs/b/Bismark/Bismark-0.24.0-GCC-12.2.0.eb

```diff
@@ -0,0 +1,41 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+# Updated to 0.23.1
+# J. Sassmannshausen NHS/GSTT
+
+easyblock = 'Tarball'
+
+name = 'Bismark'
+version = '0.24.0'
```

Comment:
version `0.24.1` has been released 3 weeks ago, can we use that?

Reply:
version is now upgraded to `0.24.1`